tests/end_to_end/job_inputs/rbt.roc

```diff
@@ -0,0 +1,53 @@
+app "build"
```

Comment:
I think it would be a good idea to make two jobs share the same dependency for this test. I think it could likely uncover a bug or two.

Reply:
you're right—I tried it and it's buggy somewhere. Gonna do some work to mitigate.

Reply:
turns out we were just dropping items that didn't have a dependency on the current item 🤷 

fixed in 3f7c6dd